qa/tasks/mgr/dashboard_v2/test_rgw.py

```diff
@@ -28,3 +27,11 @@ def test_rgw_daemon_get(self):
         self.assertIn('rgw_id', data)
         self.assertIn('rgw_status', data)
         self.assertTrue(data['rgw_metadata'])
+
+
+class RgwProxyTest(DashboardTestCase):
+
+    @authenticate
+    def test_rgw_proxy(self):
+        self._get('/api/rgw/proxy/usage')
+        self.assertJsonBody({"entries": [], "summary": []})
```

Comment:
can you also add an modifying request, like PUT, POST or DELETE?

Reply:
This is one of the reasons the PR has yet been WIP.